default_warehouse_from_sale_team/security/ir_rule_security.xml

```diff
@@ -0,0 +1,106 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo noupdate="0">
+
+    <!-- stock.picking.type -->
+    <record id="rule_default_warehouse_stock_picking_type_team" model="ir.rule">
+        <field name="name">Limited access to picking types (filtered by sale teams)</field>
+        <field name="model_id" search="[('model','=','stock.picking.type')]" model="ir.model" />
+        <field name="domain_force">[
+            ('warehouse_id', 'in', user.sale_team_ids.default_warehouse_id.ids),
+        ]</field>
+        <field
+            name="groups"
+            eval="[(6, 0, [ref('default_warehouse_from_sale_team.group_limited_default_warehouse_spt')])]"
+        />
+    </record>
+    <record id="rule_default_warehouse_stock_picking_type_all" model="ir.rule">
+        <field name="name">Access to all picking types</field>
+        <field name="model_id" search="[('model','=','stock.picking.type')]" model="ir.model" />
+        <field name="domain_force">[(1, '=', 1)]</field>
+        <field
+            name="groups"
+            eval="[(6, 0, [ref('default_warehouse_from_sale_team.group_manager_default_warehouse_spt')])]"
```

Comment:
- command syntax (separated commit)
- Superfluous module in external ID (look for other possible cases)